source/PCM_API.c

```diff
@@ -16,17 +16,23 @@
 #include "PCM_API.h"
 #include "SPI_API.h"
 
+#define write 0
```

Comment:
Ah, there i found something,
Please Write preprocessor defines in capital letters
#define WRITE 0
#define READ 1
use refactor to change...